numba/tests/test_jit_call_chains.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import print_function, division, absolute_import
+
+import math
+import numpy as np
+from numba import vectorize, guvectorize, njit, types
+from .support import unittest
+
+
+@vectorize
+def jitted_v1(alpha, beta):
+    return np.sin(alpha) + np.cos(beta)
+
+
+sig = [(types.float32[:], types.float32[:], types.float32[:]),
+       (types.float64[:], types.float64[:], types.float64[:])]
+
+
+@guvectorize(sig, '(n),()->(n)')
```

Comment:
Gufunc supports taking higher dimension array than the one given in the signature.  That feature is not implemented or tested inside njit.

For example, this gufunc can be used with:

```python
    x = np.arange(10, dtype=np.float64).reshape(2, 5)
    y = 1
    r = jitted_g1(x, y)
```